docs/src/main/sphinx/connector/faker.md

```diff
@@ -126,11 +125,13 @@ See the Datafaker's documentation for more information about
 [the expression](https://www.datafaker.net/documentation/expressions/) syntax
 and [available providers](https://www.datafaker.net/documentation/providers/).
 
-To test a generator expression, without having to recreate the table, use the
-`random_string` function from the `default` schema:
+Use the `random_string` function from the `default` schema of the `generator`
+catalog to test a generator expression:
 
 ```sql
-SELECT default.random_string('#{Name.first_name}')
+SELECT generator.default.random_string('#{Name.first_name}');
```

Comment:
Can you confirm/test this for me @nineinchnick  ..

Reply:
Catalog name is set by the user, and we don't have examples in this section, that's why I omitted it

Reply:
thats why the sentence above includes the catalog name..  without it in the example it potentially doesnt work.

Reply:
So we have to mention again that this assumes the catalog has this specific name

Reply:
Lets test this next week together (unless I get around to it earlier myself...)